G-licious/Controllers/StationsViewController.m

```diff
@@ -1,405 +0,0 @@
-//
```

Comment:
They could but I would have to restore all of them. I don't know how much work that would be but can do it if you think its best.